src/collectd.conf.pod

```diff
@@ -2699,6 +2699,26 @@ TCP-Port to connect to. Defaults to B<7634>.
 
 =back
 
+=head2 Plugin C<hugepages>
+
+To get values from B<hugepages> collectd read directories
+"/sys/devices/system/node/*/hugepages" and
+"/sys/kernel/mm/hugepages"
+Reading of these directories can be disabled by following
+options. Default is enabled.
+
+=over 4
+
+=item B<EnableNuma> I<true>|I<false>
+
+Enable/Disable reading stats in "/sys/devices/system/node/*/hugepages"
+
+=item B<EnableMM> I<true>|I<false>
```

Comment:
`EnableMM` is quite cryptic. What does this option do? What kind of metrics will I get when enabling it?